checkov/common/bridgecrew/vulnerability_scanning/image_scanner.py

```diff
@@ -152,20 +155,18 @@ def should_download(self) -> bool:
         file_age = (time.time() - last_modification.st_mtime)
         return file_age >= int(os.getenv("CHECKOV_EXPIRATION_TIME_IN_SEC", CHECKOV_SEC_IN_WEEK))
 
-    def setup_twistcli(self) -> None:
-        try:
-            if self.should_download():
-                if not os.path.exists(bridgecrew_dir):
-                    try:
-                        os.makedirs(bridgecrew_dir)
-                    except FileExistsError:
-                        # In multi-processing, this might meet a race condition
-                        pass
-                self.cleanup_scan()
-                docker_image_scanning_integration.download_twistcli(image_scanner.twistcli_path)
-        except Exception:
-            logging.error("Failed to setup docker image scanning", exc_info=True)
-            raise
+    def setup_twistcli(self) -> bool:
+        if self.should_download():
+            if not os.path.exists(bridgecrew_dir):
+                try:
+                    os.makedirs(bridgecrew_dir)
+                except FileExistsError:
+                    # In multi-processing, this might meet a race condition
+                    pass
+            self.cleanup_scan()
+            status: bool = docker_image_scanning_integration.download_twistcli(image_scanner.twistcli_path)
+            return status
```

Comment:
```suggestion
            return docker_image_scanning_integration.download_twistcli(image_scanner.twistcli_path)
```

Reply:
I thought about it, of course
I think that the second way (what you suggested) is harder to understand that it is just status rather than actual output